tests/classes/core/PKPRequestTest.php

```diff
@@ -254,7 +254,7 @@ public function testGetServerHostWithHostname() {
 		$_SERVER = array(
```

Comment:
It looks like this assignment could be removed, since we're no longer using `HOSTNAME`.